debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/YBReplicaIdentity.java

```diff
@@ -0,0 +1,57 @@
+package io.debezium.connector.postgresql.connection;
+
+import io.debezium.connector.postgresql.PostgresConnectorConfig;
+import io.debezium.relational.TableId;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+/**
+ * Helper class to assist in storing the replica identity information of a given table.
+ *
+ * @author Vaibhav Kushwaha (vkushwaha@yugabyte.com)
+ */
+public class YBReplicaIdentity {
+  private static final Logger LOGGER = LoggerFactory.getLogger(YBReplicaIdentity.class);
+
+  private final String GET_REPLICA_IDENTITY_FORMAT = "SELECT relreplident FROM pg_class WHERE oid = '%s'::regclass;";
```

Comment:
This would have to change. `pg_class` will return the latest replica identity of the table. In the current limited support, we need the replica identity of the table at the time of stream creation.

Can you read this from the `RELATION` message sent:  https://www.postgresql.org/docs/11/protocol-logicalrep-message-formats.html

Some server side changes are also needed to send the correct replica identity but you can assume that it will be correct and implement it.

Can be done in a follow-up if you wish.

Reply:
This will be read from the `RELATION` message - I will create a follow up PR to address the required changes.

Reply:
Yes, they are in. We send the correct replica identity now.